db/schema.rb.example

```diff
@@ -11,424 +10,404 @@
+    t.integer "likeable_id"
+    t.integer "user_id"
+    t.string "likeable_type"
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "emoji_type"
+    t.string "emoji_type"
   end
 
-  create_table "node", primary_key: "nid", force: true do |t|
-    t.integer "vid",                                    default: 0,  null: false
-    t.string  "type",                        limit: 32, default: "", null: false
-    t.string  "language",                    limit: 12, default: "", null: false
-    t.string  "title",                                  default: "", null: false
-    t.integer "uid",                                    default: 0,  null: false
-    t.integer "status",                                 default: 1,  null: false
-    t.integer "created",                                default: 0,  null: false
-    t.integer "changed",                                default: 0,  null: false
-    t.integer "comment",                                default: 0,  null: false
-    t.integer "promote",                                default: 0,  null: false
-    t.integer "moderate",                               default: 0,  null: false
-    t.integer "sticky",                                 default: 0,  null: false
-    t.integer "tnid",                                   default: 0,  null: false
-    t.integer "translate",                              default: 0,  null: false
-    t.integer "cached_likes",                           default: 0
-    t.integer "comments_count",                         default: 0
-    t.integer "drupal_node_revisions_count",            default: 0
-    t.string  "path"
+  create_table "node", primary_key: "nid", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+    t.integer "vid", default: 0, null: false
+    t.string "type", limit: 32, default: "", null: false
+    t.string "language", limit: 12, default: "", null: false
+    t.string "title", default: "", null: false
+    t.integer "uid", default: 0, null: false
+    t.integer "status", default: 1, null: false
+    t.integer "created", default: 0, null: false
+    t.integer "changed", default: 0, null: false
+    t.integer "comment", default: 0, null: false
+    t.integer "promote", default: 0, null: false
+    t.integer "moderate", default: 0, null: false
+    t.integer "sticky", default: 0, null: false
+    t.integer "tnid", default: 0, null: false
+    t.integer "translate", default: 0, null: false
+    t.integer "cached_likes", default: 0
+    t.integer "comments_count", default: 0
+    t.integer "drupal_node_revisions_count", default: 0
+    t.string "path"
     t.integer "main_image_id"
-    t.string  "slug"
-    t.integer "legacy_views",                           default: 0
-    t.integer "views",                                  default: 0
+    t.string "slug"
+    t.integer "legacy_views", default: 0
+    t.integer "views", default: 0
+    t.decimal "latitude", precision: 20, scale: 17
```

Comment:
OK, sorry i have been traveling, but this and the line below I think are the only lines needed except for the timestamp at top. The rest is mostly whitespace changes as well as some encoding changes we don't actually want to modify in this PR. Thanks, Camila!

Reply:
No problem :)

Is it ok now?